LLMObs setup update (#23149)
DataDog/documentation: content/en/tracing/llm_observability/sdk.md

Enable LLM Observability through the `LLMOBs.enable()` function, as shown in the example below.

> **Note**: If you do not have Datadog APM set up, set `dd_llmobs_no_apm` to `True`. This configures the `ddtrace` library to not send any data that requires Datadog APM.

Reviewer: ```suggestion
**Note**: If you do not have Datadog APM set up, set `dd_llmobs_no_apm` to `true`. This configures the `ddtrace` library to not send any data that requires Datadog APM.
```

> **Note**: `LLMObs.enable()` automatically turns automatic tracing for support LLM Observability integrations (Bedrock, OpenAI, Langchain). To only enable a certain subset of these integrations, use the `integrations` argument.

Reviewer: ```suggestion
**Note**: `LLMObs.enable()` turns on automatic tracing for supported LLM Observability integrations (Bedrock, OpenAI, Langchain). To only enable it for a subset of these integrations, use the `integrations` argument.
```
